### PR TITLE
[R120-4] Use patched log-warper to set utf8 encoding

### PR DESCRIPTION
**Pull request by parsonsmatt**
## Description

Unicode characters cause the API endpoints to die. This PR uses a patched version of `log-warper` that I have branched off of the current version that we are using: https://github.com/input-output-hk/log-warper/pull/2 is the PR, though it should probably live in it's own branch, as the paths have diverged a bit.

I suggest we get the branch into `input-output-hk/log-warper` so that we can more rapidly iterate on that while moving to `katip`.

## Linked issue

https://iohk.myjetbrains.com/youtrack/issue/R120-4#comment=93-20912

## Type of change

- [x] 🐞 Bug fix (non-breaking change which fixes an issue)
- [~] 🛠 New feature (non-breaking change which adds functionality)
- [~] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)
- [~] 🏭 Refactoring that does not change existing functionality but does improve things like code readability, structure etc
- [~] 🔨 New or improved tests for existing code
- [~] ⛑ git-flow chore (backport, hotfix, etc)

## Developer checklist

- [x] I have read the [style guide](https://github.com/input-output-hk/cardano-sl/blob/develop/docs/style-guide.md) document, and my code follows the code style of this project.
- [x] If my code deals with exceptions, it follows the [guidelines](https://github.com/input-output-hk/cardano-sl/blob/develop/docs/exceptions.md).
- [x] I have updated any documentation accordingly, if needed. Documentation changes can be reflected in opening a PR on [cardanodocs.com](https://github.com/input-output-hk/cardanodocs.com), amending the inline [Haddock](https://www.haskell.org/haddock/) comments, any relevant README file or one of the document listed in the [docs](https://github.com/input-output-hk/cardano-sl/tree/develop/docs) directory.

## Testing checklist

- [~] I have added tests to cover my changes.
- [~] All new and existing tests passed.

### CODE REVIEW

File: wallet-new/integration/Util.hs [hunk 1]
```diff
@@ -94,3 +94,9 @@ shouldPrism a b = do
     pure x
 
 infixr 8 `shouldPrism`
+
+shouldPrism_ :: Show s => s -> Prism' s a -> IO ()
+shouldPrism_ a b =
+    a `shouldSatisfy` has b
+
+infixr 8 `shouldPrism_`
```
[the remark this answers is not in the document]
Comment (parsonsmatt) [reply wallet-new/integration/Util.hs]: I'm a fan of giving explicit names to lens-y stuff, especially if it hides the lensy-ness of it

File: pkgs/default.nix [hunk 2]
```diff
@@ -24855,9 +24855,9 @@ inherit (pkgs) which;};
            pname = "log-warper";
            version = "1.8.0";
            src = fetchgit {
-             url = "https://github.com/serokell/log-warper.git";
-             sha256 = "1rqpkid2c58pk813kb7wjcibsxf97cn7vjf3bv3dma1k5b818ana";
-             rev = "7f95c6990ef93a289678755cb1db71c2a4ecdfb5";
+             url = "https://github.com/parsonsmatt/log-warper.git";
+             sha256 = "0izivirmg5vclc0mj7rxr1vvdwgv7vc7sll3pgdjxxbajr93jrbk";
+             rev = "66ae935fdf47d1997d95dcc60cee0637a2eb74b1";
```
[the remark this answers is not in the document]
Comment (parsonsmatt) [reply pkgs/default.nix]: Sounds good!

Right now, what I think we should do is replicate that branch on `input-output-hk` repository. But I don't have access to that repository to do that.

[the remark this answers is not in the document]
Comment (parsonsmatt) [reply pkgs/default.nix]: Right now, no one actually knows who is in charge of that repo? :\

**Comment by parsonsmatt** [comment]
I am unable to replicate the bug, or fix, on my computer. Even doing `LC_ALL=C.POSIX` does not trigger it for me :\

**Comment by disassembler** [comment]
rebased on develop to clear appveyor failure.

**Pull request merged**